Alignment/CommonAlignmentProducer/python/ALCARECOTkAlMuonIsolatedHI_cff.py

```diff
@@ -1,27 +1,21 @@
-# AlCaReco for track based alignment using isolated muon tracks
+# AlCaReco for track based alignment using isolated muon tracks - relaxed cuts for PbPb collisions
 import FWCore.ParameterSet.Config as cms
 
 from Alignment.CommonAlignmentProducer.ALCARECOTkAlMuonIsolated_cff import *
 
 ALCARECOTkAlMuonIsolatedHIHLT = ALCARECOTkAlMuonIsolatedHLT.clone(
-    eventSetupPathsKey = 'TkAlMuonIsolated'
+    eventSetupPathsKey = 'TkAlMuonIsolatedHI'
     )
 
 ALCARECOTkAlMuonIsolatedHIDCSFilter = ALCARECOTkAlMuonIsolatedDCSFilter.clone()
 
-ALCARECOTkAlMuonIsolatedHIGoodMuons = ALCARECOTkAlMuonIsolatedGoodMuons.clone()
-ALCARECOTkAlMuonIsolatedHIRelCombIsoMuons = ALCARECOTkAlMuonIsolatedRelCombIsoMuons.clone(
-    src = 'ALCARECOTkAlMuonIsolatedHIGoodMuons'
-)
-
 ALCARECOTkAlMuonIsolatedHI = ALCARECOTkAlMuonIsolated.clone(
```

Comment:
Ah, that was a stupid mistake that I should have caught before committing.  I made a fix and I'm testing locally before I update the branch.